src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.csproj

```diff
@@ -6,17 +6,25 @@
          to its own obsolete API. -->
     <NoWarn>618</NoWarn>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <AssemblyVersion>4.0.21.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
     <NuGetTargetMoniker>.NETPlatform,Version=v5.4</NuGetTargetMoniker>
+    <ProjectGuid>{B17014F1-D902-417F-89B0-271204695831}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Runtime.InteropServices.cs" />
-    <Compile Include="System.Runtime.InteropServices.Manual.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../System.Runtime.InteropServices.PInvoke/ref/System.Runtime.InteropServices.PInvoke.csproj">
+      <Private>false</Private>
+    </ProjectReference>
+    <ProjectReference Include="../../System.Runtime/ref/System.Runtime.csproj">
```

Comment:
Why does this need to be a project reference to System.Runtime? It should just be a package reference in project.json.